### PR TITLE
feat(react-components): mostly standardize card paddings

### PR DESCRIPTION
**Pull request by jeysal**
and remove naming / story inconsistencies in the profile cards

### CODE REVIEW

File: apps/storybook/src/ProfileSkills.stories.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { array } from '@storybook/addon-knobs';
-import { SkillsSection } from '@asap-hub/react-components';
+import { ProfileSkills } from '@asap-hub/react-components';
```
Comment (fampinheiro) [inline apps/storybook/src/ProfileSkills.stories.tsx:3]: Not sure about this name change. This component is used on Profile and Teams.

Comment (jeysal) [reply apps/storybook/src/ProfileSkills.stories.tsx:3]: Hmm, I wasn't aware of that.
What do you think of wording everything that is specific to users `UserProfile`, everything that is specific to teams `TeamProfile` and everything that is for both `Profile`?

Comment (jeysal) [reply apps/storybook/src/ProfileSkills.stories.tsx:3]: (in another PR ofc)

**Comment by jeysal** [comment]
rebased

**Review by fampinheiro — APPROVED** [review]

**Comment by jeysal** [comment]
patch diff check complains about a line in `TeamAbout` where only a variable name change happened, ignoring

**Pull request merged**